test/polling/infrequent_polling_test.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'test'
+require 'securerandom'
+require 'temporalio/testing'
+require 'temporalio/worker'
+require 'polling/infrequent/greeting_workflow'
+require 'polling/infrequent/compose_greeting_activity'
+require 'polling/infrequent/test_service'
+
+module Polling
+  class InfrequentPollingTest < Test
+    def test_workflow_completes_after_polling
+      task_queue = "tq-#{SecureRandom.uuid}"
+
+      Temporalio::Testing::WorkflowEnvironment.start_time_skipping do |env|
```

Comment:
Not sure if all environments will support time skipping. In our Ruby SDK base `Test` class we have:

```
  def skip_if_not_x86!
    skip('Test only supported on x86') unless RbConfig::CONFIG['host_cpu'] == 'x86_64'
  end
```

And we invoke that at the top of any test that needs time skipping. May want to do the same here.